specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -617,10 +614,10 @@ if libraries need to access classes packaged in the application modules.
 Components in the web container must have
 access to the following classes
 
-and resources. Note that as of Java EE 6, EJB
-components may be packaged in a web component module. Such EJB
+and resources. Note that as of Java EE 6, Jakarta Enterprise Beans
```

Comment:
Here the context is clearly before Java EE 8/Jakarta EE 8...  We don't need to modify every instance of EJB...

Reply:
done